config/crd/bases/eventing.keda.sh_cloudeventsources.yaml

```diff
@@ -52,6 +52,18 @@ spec:
                     - uri
                     type: object
                 type: object
+              eventSubscription:
+                description: EventSubscription defines filters for events
+                properties:
+                  excludedEventTypes:
```

Comment:
Let's make sure we have validation on the event types; if they specify a type that is not supported then block creation

Reply:
Good point. Will add the validation.

Reply:
Is there a way we can have them in the CRD or not really?